packages/client/src/getFees.ts

```
@@ -1,84 +1,119 @@
+} from "../../../types/getFees.types";
 import { ZetaChainClient } from "./client";
 
-const fetchZEVMFees = async function (
-  zrc20: any,
+const fetchZEVMFees = async (
+  zrc20: (typeof mainnet)[number],
   rpcUrl: string,
-  foreignCoins: any
-) {
+  foreignCoins: ForeignCoin[]
+): Promise<FeeItem | void> => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)